src/lib.rs

```diff
@@ -311,9 +311,13 @@ pub unsafe extern "C" fn wgpuSurfaceGetPreferredFormat(
     surface: id::SurfaceId,
     adapter: id::AdapterId,
 ) -> native::WGPUTextureFormat {
```

Comment:
Keeping this function is fine - to keep compliant with webgpu.h - but we should also have a function which returns all the formats so the user can choose.

Reply:
added `wgpuSurfaceGetSupportedFormats`